test-network-function/tnf_config.yml

```diff
@@ -25,7 +25,10 @@ targetPodLabels:
 #       namespace: default
 #       subscriptionName: etcd
 #       autogenerate: false
-
+#   deploymentsUnderTest:
+#     - name: test
+#       namespace: tnf
+#       replicas: 2
```

Comment:
I don't think we should specify the replicas in the example.
```
 oc get deployment
NAME      READY   UP-TO-DATE   AVAILABLE   AGE
partner   1/1     1            1           24h
test      2/2     2            2           24h
➜  go 

```
tnf has replicas of 2. it also can be read from ``oc describe deployment tnf``
so it something that should be left to autodiscovery.

Reply:
As discussed with Jun, we just want to include this as a complement to the autodiscovery mechanism in order to create the deployments list.